test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/Scene3DPanelTest.cs

```diff
@@ -0,0 +1,134 @@
+                Size = new Size(100, 100),
+                Position = new Position(25, 100),
+            };
+            parent.Add(child);
+
+            panel = new Panel()
+            {
+                Position = new Position(0.0f, 0.0f, 0.5f),
+                Size = new Size(2.0f, 1.0f),
+                PanelResolution = new Vector2(400, 500),
+                Content = parent,
+                BackFacePlaneColor = backFaceColors[colorIndex],
+            };
+            sceneView.Add(panel);
+
+            SetGuide();
+
+            animation = new Animation(15000);
+            animation.AnimateTo(panel, "orientation", new Rotation(new Radian(new Degree(180.0f)), Vector3.YAxis));
+            animation.AnimateTo(panel, "size", new Vector3(1.0f, 2.0f, 0.0f));
+            animation.Looping = true;
+            animation.LoopingMode = Animation.LoopingModes.AutoReverse;
+            animation.Play();
+
+            window.KeyEvent += (s, e) =>
+            {
+                if (e.Key.State == Key.StateType.Down)
+                {
+                    if (e.Key.KeyPressedName == "1")
+                    {
+                        panel.Transparent = !panel.Transparent;
+                    }
+                    else if (e.Key.KeyPressedName == "2")
+                    {
+                        panel.DoubleSided = !panel.DoubleSided;
+                    }
+                    else if (e.Key.KeyPressedName == "3")
+                    {
+                        panel.UsingBackFacePlane = !panel.UsingBackFacePlane;
+                    }
+                    else if (e.Key.KeyPressedName == "4")
+                    {
+                        colorIndex = ((colorIndex + 1) % 3);
+                        panel.BackFacePlaneColor = backFaceColors[colorIndex];
+                    }
+                    SetGuide();
+                }
+            };
+        }
+
+        void SetGuide()
+        {
+            if(guideText == null)
+            {
+                guideText = new TextLabel()
+                {
+                    PositionUsesPivotPoint = true,
+                    PivotPoint = PivotPoint.BottomLeft,
+                    ParentOrigin = ParentOrigin.BottomLeft,
+                    PointSize = 30.0f,
+                    MultiLine = true,
+                };
+            }
+            window.Add(guideText);
+
+            string backFaceColor = string.Format($"({panel.BackFacePlaneColor.R}, {panel.BackFacePlaneColor.G}, {panel.BackFacePlaneColor.B})");
+            string guide = string.Format($"Press 1 to convert transparency - Current value : {panel.Transparent}\nPress 2 to convert double sided - Current value : {panel.DoubleSided}\nPress 3 to convert back face - Current value : {panel.UsingBackFacePlane}\nPress 4 to convert back face color - Current value : {backFaceColor}\n");
+            guideText.Text = guide;
+        }
+
+        public void Deactivate()
+        {
```

Comment:
Plea
```suggestion
        {
            sceneView?.Dispose();
```

NUI Sameple doeant unparent vies automatically. Please unparent root view what you added here

Reply:
Thank you.